frontend/src/app/components/containers/complaints/complaint-list.tsx

```diff
@@ -19,6 +19,7 @@ import { selectDefaultPageSize, selectDefaultZone } from "../../../store/reducer
 import { WildlifeComplaintListItem } from "./list-items/wildlife-complaint-list-item";
 import { AllegationComplaintListItem } from "./list-items/allegation-complaint-list-item";
 import ComplaintPagination from "../../common/complaint-pagination";
+import { Alert } from "react-bootstrap";
```

Comment:
I don't believe this import is required.

Reply:
Ran a fixup, cleaned up the vestigial import from [c4d44fe](https://github.com/bcgov/nr-compliance-enforcement/pull/716/commits/c4d44fe64e7559238864d34b563adb460221e4b3) leftover from my testing.